schematron/v2.2.0/tests/data/HVAC_example1.xml

```diff
@@ -138,6 +138,12 @@
                                         <auc:ReheatSource>Heating plant</auc:ReheatSource>
                                         <auc:ReheatControlMethod>Single Maximum</auc:ReheatControlMethod>
                                         <auc:ReheatPlantID IDref="HeatingPlant-A" />
+                                        <auc:FanBased>
+                                            <auc:AirSideEconomizer ID="AirSideEconomizer-A">
+                                                <auc:AirSideEconomizerType>None</auc:AirSideEconomizerType>
+                                                <auc:EconomizerControl>Fixed</auc:EconomizerControl>
```

Comment:
I believe for an `auc:AirSideEconomozerType='None'` we would not expect an `auc:EconomizerControl` element.  @nllong ??

Reply:
correct, if `AirSideEconomizerType` is `None`, then there should not be an economizer. That assumes that `EconomizerControl` is only air side, right?

Reply:
I guess, in this context: `auc:Delivery/auc:DeliveryType/auc:CentralAirDistribution/auc:AirDeliveryType/auc:FanBased`, the assumption is that we are just talking about airside economizer.  Other `auc:EconomizerControl` might exist, just not here.

Reply:
Summary is that if auc:AirSideEconomozerType != 'None' then there MUST be an auc:EconomizerControl. We won't do any checks that when auc:AirSideEconomozerType == 'None' there is not an auc:EconomizerControl

Reply:
Fixed here: d5c3ec7b37a929e21204f48c3c6d30c72ae4d96c